docs-app/package.json

```diff
@@ -36,7 +36,7 @@
     "@ember/string": "^3.0.1",
     "@ember/test-helpers": "^3.1.0",
     "@embroider/compat": "^2.0.0",
-    "@embroider/core": "^2.0.0",
+    "@embroider/core": "2.0.2",
```

Comment:
Had to pin this for now due to an error I was seeing.  The task to upgrade us to the latest embroider stuff is at https://github.com/CrowdStrike/ember-toucan-core/issues/210

The error was:

```bash
docs-app:build: $TMPDIR/embroider/6cc172/node_modules/.pnpm/file+packages+ember-toucan-core_@babel+core@7.20.12_@crowdstrike+ember-toucan-styles@2.0.1_@e_y4sbboigrbv5roskhqf7pymkbu/node_modules/@crowdstrike/ember-toucan-core/dist/components/form/fields/textarea.js: Scope objects for `precompileTemplate` may only contain direct references to in-scope values, e.g. { Field } or { Field: Field }
docs-app:build:   115 |   `, {
docs-app:build:   116 |   strictMode: true,
docs-app:build: > 117 |   scope: () => ({
docs-app:build:       |   ^
docs-app:build:   118 |     Field: ToucanFormFieldComponent,
docs-app:build:   119 |     hash,
docs-app:build:   120 |     field,
```

Reply:
Mind adding a comment somewhere? One possibility:

```
`"devDependencyComments": {
  "@embroider/core": "[your comment]"
},`